index.js

```diff
@@ -24,7 +31,14 @@
 // ***** Question 3 *****
 // *** Uncomment the lines below to test
 // console.log("%cQuestion 3", "color: red") 
-
+// let currentThirstLevel = 12
+
+// function drinkWater(currentThirstLevel) {
+//   console.log(`Man I sure am thirsty`)
+//   currentThirstLevel -= 1
+//   console.log(`Ahh that hits the spot`)
+//   return currentThirstLevel
+// }
```

Comment:
Nice work remembering that we need to explicitly return values in JS.

As an additional exercise, how would this look as an arrow function?